src/com/company/Contact.java

```diff
@@ -0,0 +1,23 @@
+package com.company;
+
+public class Contact {
+    String name;
+    String telephone;
+    String email;
```

Comment:
@SFDC-Melby-Arguedas Kee in mind Encapsulation, this attributes could be private